testcontainers/src/core/container.rs

```diff
@@ -182,18 +183,23 @@ where
         .unwrap_or_else(|_| panic!("container {} has missing or invalid bridge IP", self.id))
     }
 
-    pub fn exec(&self, cmd: ExecCommand) {
+    pub fn exec(&self, cmd: ExecCommand) -> ExecOutput {
```

Comment:
Can you replicate this API for the `ContainerAsync` type please?

Reply:
`DockerAsync` doesn't have `exec`. Should I add this?

Reply:
> `ContainerAsync` & `DockerAsync` doesn't have `exec`. Should I add this? 

Oh okay. We really need some tests to keep these APIs on par.

All good then, can you open an issue? Thanks!

Reply:
Sure, created #475